### DISCUSSION TITLE
✨ OnlyOffice (Community)

### DISCUSSION THREAD

**Discussion by JigSawFr** [runtipi/runtipi-appstore#2215] [Requests]
### Overview
ONLYOFFICE Community Server is a free open-source collaborative system developed to manage documents, projects, customer relationship and email correspondence, all in one place.

Starting from version 11.0, Community Server is distributed as ONLYOFFICE Groups on terms of Apache License.

### Functionality

- Cross platform solution: Linux, Windows
- Document management
- Integration with Google Drive, Box, Dropbox, OneDrive, OwnCloud
- File sharing
- Document embedding
- Access rights management
- Customizable CRM
- Web-to-lead form
- Invoicing system
- Project Management
- Gantt Chart
- Milestones, task dependencies and subtasks
- Time tracking
- Automated reports
- Blogs, forums, polls, wiki
- Calendar
- Email Aggregator
- People module (employee database)
- Support of more than 20 languages

### Official Website

http://www.onlyoffice.com/

### Documentation link

https://helpcenter.onlyoffice.com/

### Last application release & date

12.5.2 (26/05/2023)

### Application license

Apache-2.0 license

### Source code repository link

https://github.com/ONLYOFFICE/CommunityServer

### Docker image link

https://helpcenter.onlyoffice.com/installation/docs-community-install-docker.aspx?_ga=2.51711023.782359554.1594636128-1157782750.1587541027

### Other's

https://github.com/ONLYOFFICE/Docker-DocumentServer

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote 👍 this discussion to show your interest in this request! 

Thanks ⛺

**Comment by JigSawFr** [comment]
Originally requested by @Kchnov

**Comment by jasonpearce** [comment]
If helpful, here is OnlyOffice packaged for YunoHost:

https://github.com/YunoHost-Apps/onlyoffice_ynh

**Comment by viniciusnevescosta** [comment]
There's a chance to add it to Runtipi repo anyway?
